mattermost/mattermost-webapp
# [MM-48029] add notify admin for work template integrations

**URL for testing : https://mattermost-server-pr-22007.test.mattermost.cloud/** 
**admin userid: user-mattermost-server-pr-22007@example.mattermost.com**
**password : Cws@10102020**

#### Summary
notify admin for plugin install from the work templates.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-48029

#### Related Pull Requests
- Has server changes (https://github.com/mattermost/mattermost-server/pull/22007)

#### Screenshots
![Screenshot 2023-01-26 at 16 01 45](https://user-images.githubusercontent.com/841955/214855098-65b4971b-ae79-4be2-bc7c-937747e9e71b.png)

#### QA Test Steps
admin userid:password : user-mattermost-server-pr-22007@example.mattermost.com:Cws@10102020
- Visit https://mattermost-server-pr-22007.test.mattermost.cloud with the admin role.
- Go to the bot channel and view the notify admin post. This notification was triggered by a system user by visiting 'work template' -> 'preview section' -> 'integrations'. 
- Create and login with a new user in incognito or other browser. Go to worktemplates integrations. You should see 'Notify admin to install integrations' link. On clicking, it should say 'Admin notified'. Go to the tab with admin logged in, after a couple of minutes you could see the post appearing. 
- Try notifying the admin again by the same user and for same category and it wouldn't be possible.

#### Release Note
```release-note
NONE
```
